src/bp/core/logger/logger.ts

```diff
@@ -46,6 +47,7 @@ export class PersistedConsoleLogger implements Logger {
   private willPersistMessage: boolean = true
   private emitLogStream = true
   private serverHostname: string = ''
+  private event?: IO.Event
```

Comment:
I'd make it a `WeakRef<IO.Event>` for the sake of proper garbage collection

Reply:
Not available yet, node 13. I've added a cleanup at the end tho